db-manager/db_manager/__main__.py

```diff
@@ -0,0 +1,239 @@
+import argparse
+import logging
+import re
+import os
+from typing import List, Tuple, Iterable, Any
+from contextlib import contextmanager
+from itertools import groupby
+
+import psycopg2
+from natsort import natsorted
+
+from . import dimCountry, dimManuscript
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class FileTypePatterns:
+    COUNTRY = 'country'
+    MANUSCRIPT = 'manuscripts'
+    MANUSCRIPT_VERSION = 'versions'
+    MANUSCRIPT_STAGE = 'stages'
+
+
+class SubCommands:
+    TEARDOWN = 'teardown'
+    CREATE = 'create'
+    IMPORT_DATA = 'import-data'
+
+
+TIMESTAMP_PATTERN = r'(\d+).*'
+
+
+StagingModule = Any
```

Comment:
If we used classes it would could do better type checking (it might also make it easier to understand if we passed instances around instead of modules).